src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -12,10 +12,73 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
             + "Example: " + COMMAND_WORD;
 
-    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+    public static final String SHOWING_HELP_MESSAGE = "Here is a list of commands available, "
+            + " for more information refer to https://ay2021s2-cs2103-w16-3.github.io/tp/UserGuide.html\n"
+            + "Adding contacts: "
+            + "add NAME [-p PHONE_NUM] [-e EMAIL] [-a ADDRESS] [-t TAG]…\u200B [-b BIRTHDAY]\u200B\n"
+            + "Clearing all data: clear\n"
+            + "Deleting contacts: "
+            + "delete INDEX [INDEX…]\n"
+            + "Editing contacts: "
+            + "edit INDEX [-n NAME] [-p PHONE_NUMBER] [-e EMAIL] [-a ADDRESS] [-t TAG]…\u200B [-b BIRTHDAY]\n"
```

Comment:
Missing `exit` command.

```suggestion
            + "edit INDEX [-n NAME] [-p PHONE_NUMBER] [-e EMAIL] [-a ADDRESS] [-t TAG]…\u200B [-b BIRTHDAY]\n"
            + "Exiting application: "
            + "exit\n"
```

Reply:
Added, thanks